<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageHabitsDialog</class>
 <widget class="QDialog" name="ManageHabitsDialog">
  <property name="windowTitle">
   <string>Manage Habits</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutMain">
   <item>
    <widget class="QTableWidget" name="tblHabits">
     <property name="columnCount">
      <number>3</number>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <widget class="QPushButton" name="btnAddHabit">
       <property name="text">
        <string>Add Habit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEditHabit">
       <property name="text">
        <string>Edit Habit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveHabit">
       <property name="text">
        <string>Remove Habit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
